# Define the publishing rate, must be positive.
publish_rate: 5

# Define the frames we care about. Each frame is associated with a pose topic
# (mandatory), and optionally a parent frame (default=world) and
# initial_position (default=None).
frames:
    ball: 
        pose_topic: /catch/ball_pose
        parent_frame: world
